app.py

```diff
@@ -338,6 +338,21 @@ def report_error():
 
     return 'logged'
 
+@app.route('/client_exception', methods=['POST'])
+def report_client_exception():
+    post_body = request.json
+
+    parse_logger.log ({
```

Comment:
Why is this going into the `parse_logger`? That's the log stream Felienne does her program analysis on--is it the right place?

I would log it into the querylog logger.

Reply:
Great catch!